src/main/java/com/exasol/adapter/dialects/BaseQueryRewriter.java

```diff
@@ -97,9 +83,16 @@ protected ExaConnectionInformation getConnectionInformation(final ExaMetadata ex
         return exaConnectionInformation;
     }
 
-    private String generatePushdownSql(final String columnDescription, final String connectionDefinition,
-            final String pushdownSql) {
-        return "IMPORT INTO (" + columnDescription + ") FROM JDBC " + connectionDefinition + " STATEMENT '"
-                + pushdownSql.replace("'", "''") + "'";
-    }
+    /**
+     * Generate a query to be execute in the Exasol database, that wraps the passed query to be executed in an external
```

Comment:
```suggestion
     * Generate a query to be executed in the Exasol database, that wraps the passed query to be executed on an external
```

"Wrapping" is not always true: `IS_LOCAL`  create a `SELECT`, in rare cases you have `SELECT ... FROM VALUES`.